spec/core_functions/color/complement.hrx

```diff
@@ -88,8 +88,8 @@ Error: Missing argument $color.
   |       ^^^^^^^^^^^^ invocation
   '
   ,--> sass:color
-1 | @function complement($color) {
-  |           ================== declaration
+1 | @function complement($color, $space: null) {
```

Comment:
@mirisuzanne For functions like this that don't have similar wpt tests, is there a way to test what values they return now?

Reply:
@dvdherron we could use something like colorjs.io to mimic the function logic, and take results from that. It doesn't have the same function, but  it should have all the features we would need. (I used it sometimes while writing the spec).

I've also mocked up some functions (and tests) in https://github.com/oddbird/sass-colors-sample which I forgot about. I'll add links to both in the trello card.